hs_core/hydroshare/resource.py

```diff
@@ -77,7 +77,11 @@ def get_science_metadata(pk):
     Exceptions.NotFound  - The resource identified by pid does not exist
     Exception.ServiceFailure  - The service is unable to process the request
     """
-    return get_system_metadata(pk)
+    res = utils.get_resource_by_shortkey(pk)
+    if res:
+        return res.metadata.get_xml()
+
+    #return get_system_metadata(pk)
```

Comment:
its already in a branch until its read, so what does the extra build step save you? It also creates a local environment which a change (the overwritten Dockerfile) that someone could accidentally commit. That doesn't sound like a good idea.